src/components/HomePage.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import newsApi from './api/newsApi';
+
+const HomePage = () => {
+	const [articlesList, setArticlesList] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		getArticles().then((response) => {
+			setArticlesList(response.data.articles);
+			setIsLoading(false);
+		});
+	}, []);
+
+	if (isLoading) {
+		return <div className="d-flex align-items-center justify-content-center">Loading ...</div>;
+	}
+
+	function getArticles() {
+		return newsApi.get('/articles');
```

Comment:
Why do you need this when you have this already established in your newsApi.js file?